connector/src/main/java/org/geysermc/connector/network/translators/bedrock/BedrockAnimateTranslator.java

```diff
@@ -37,19 +37,26 @@
 @Translator(packet = AnimatePacket.class)
 public class BedrockAnimateTranslator extends PacketTranslator<AnimatePacket> {
 
+    private boolean is_steering_left;
+    private boolean is_steering_right;
```

Comment:
Variable name formatting should be in camelCase, not snake_case, and this data should be stored in `GeyserSession`.